src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -320,6 +328,99 @@ def test_api_video_read_list_token_user(self):
         )
         self.assertEqual(response.status_code, 404)
 
+    @override_settings(CLOUDFRONT_SIGNED_URLS_ACTIVE=False)
+    def test_api_video_with_a_thumbnail(self):
+        """A video with a custom thumbnail should have it in its payload."""
+        video = VideoFactory(
+            pk="38a91911-9aee-41e2-94dd-573abda6f48f",
+            uploaded_on=datetime(2018, 8, 8, tzinfo=pytz.utc),
+            upload_state="ready",
+        )
+        thumbnail = ThumbnailFactory(
+            video=video,
+            uploaded_on=datetime(2018, 8, 8, tzinfo=pytz.utc),
+            upload_state="ready",
+        )
+
+        jwt_token = AccessToken()
+        jwt_token.payload["video_id"] = str(video.id)
+        jwt_token.payload["roles"] = ["instructor"]
+
+        # Get the video linked to the JWT token
+        response = self.client.get(
+            "/api/videos/{!s}/".format(video.id),
+            HTTP_AUTHORIZATION="Bearer {!s}".format(jwt_token),
+        )
+        self.assertEqual(response.status_code, 200)
+        content = json.loads(response.content)
+
+        self.assertEqual(
+            content,
```

Comment:
maybe you only need to test the value of content["thumbnail"] if the rest is already tested? That what you announced in your docstring.